x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/actions/SearchableSnapshotActionIT.java

```diff
@@ -188,4 +196,149 @@ public void testDeleteActionDeletesSearchableSnapshot() throws Exception {
         }, 30, TimeUnit.SECONDS));
     }
 
+    public void testCreateInvalidPolicy() throws Exception {
+        String snapshotRepo = randomAlphaOfLengthBetween(4, 10);
+        createSnapshotRepo(client(), snapshotRepo, randomBoolean());
```

Comment:
I don't think we need to actually create the repository here, since the test doesn't take a snapshot?